create-ladder.js

```diff
@@ -74,6 +82,20 @@
     })
   }
 
+  //Slider
+
+  const seekslider = document.getElementById("slider");
+
+  seekslider.addEventListener('input', function() {
+    const v = this.value;
+    seekslider.style.background = "-moz-linear-gradient(left,  #3B3561 0%, #3B3561 "+ v +"%, white "+ v +"%, white 100%)";
+    seekslider.style.background = "-webkit-gradient(linear, left top, right top, color-stop(0%,#3B3561), color-stop("+ v +"%,#3B3561), color-stop("+ v +"%,white), color-stop(100%,white))";
+    seekslider.style.background = "-webkit-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "-o-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "-ms-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "linear-gradient(to right,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
```

Comment:
This is javascript, not css. the was css cascades is different to how assignment works in js.

This code is the same as the following css
```css
#slideslider {
  background: linear-gradient(to right,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%);
}
```